kratos/solving_strategies/builder_and_solvers/residualbased_block_builder_and_solver.h

```diff
@@ -263,8 +259,8 @@ class ResidualBasedBlockBuilderAndSolver
      */
     void BuildLHS(
         typename TSchemeType::Pointer pScheme,
-        ModelPart& rModelPart,
-        TSystemMatrixType& A) override
+        ModelPart &rModelPart,

```

Comment:
Moving the & is unnecessary and it is an aesthetic change (usually if I do that I get blocked). When we decided to move the &? BTW